### ISSUE TITLE
Improve banner settings

### ISSUE THREAD

**Issue by thisisdano** [uswds/uswds#3509]
Users should be able to set the background color of the banner in settings, and our new, smarter color contrast tool should allow reliable color changing.

Perhaps there are other banner-related settings that would be useful?
